### ISSUE TITLE
Game crashing on Loading screen (after press on Story)

### ISSUE THREAD

**Issue by emozio666** [LukeRoss00/gta5-real-mod#382]
Hi. After another update of the game, I cannot play the game in VR. Game crashes on loading story mode. 
(I must say right away that I have been playing your mod for the second year and this is the best thing that happened to me in VR.)
My setup;

Script Hook V: Released | 23 Jul 2021
CPU: i7 10700K
GPU: 2070 RTX SUPER
VR headset model: HTC VIVE
game version and build: v 1.0.2372.0
release of the mod: r7
did you follow all the Quick setup steps carefully, in particular step 4? - Yes
which settings did you choose (H/M/L) when running the batch file RealConfig.bat? - i tried  every setup
did the batch file report any errors? - no
did you change the graphics settings manually? - no
are you using any other mods? - yes (NaturalVision Remastered - have been playing with this mod since day one vr mod.)

**Comment by LukeRoss00** [comment]
Hi @emozio666 and @rikiwelche, thanks for the kind words!

I'm not sure what the problem is. I have the game updated to 1.0.2372.0 (Steam version) on my PC and it loads Story Mode and plays in VR just fine. No crashes.

Are you sure you updated Script Hook V correctly? Could you attach your `ScriptHookV.log` file?

**Comment by emozio666** [comment]
[ScriptHookV.log](https://github.com/LukeRoss00/gta5-real-mod/files/6880590/ScriptHookV.log)

**Comment by rikiwelche** [comment]
hello luc, thank you for your answer,
here is my scriptthook.log file
Personally I use an rtx2060 with the natural vision envolved mode. The set has also always worked well for me, even after every update of the mode every month, it is since this last update of the game and of scripthook. I have the version with rockstar launcher. 
[ScriptHookV.log](https://github.com/LukeRoss00/gta5-real-mod/files/6883779/ScriptHookV.log)

**Comment by LukeRoss00** [comment]
Something is going wrong with your Script Hook V, and I have no idea what it could be. It shouldn't depend on the mod, because the problem happens before my `RealVR.asi` file is even loaded. See how the log looks for me:

```
// GTA V SCRIPT HOOK (build Jul 23 2021, v1.0.2372.0)
//     (C) Alexander Blade 2015-2021
[20:52:31] INIT: Started
[20:52:31] INIT: Success, game version is VER_1_0_2372_0_STEAM
[20:52:31] INIT: Registering script 'NativeTrainer.asi' (0x00007FFD10D8A3C0)
[20:52:31] INIT: Registering script 'RealVR.asi' (0x00007FFD0242A580)
[20:52:39] INIT: Pool 1 extended
[20:52:39] INIT: Pool 2 extended
[20:52:39] INIT: Pool 3 extended
[20:52:39] INIT: Pool 4 extended
[20:52:44] INIT: GtaThread collection size 189
[20:52:44] INIT: wnd proc 0x00000000FFFF06CF
[20:52:44] INIT: IDXGISwapChain 0x00007FFCD8C8D990 (0x00007FFCD8A75920)
[20:52:44] INIT: IDXGISwapChain set
[20:52:44] INIT: DX init started
[20:52:44] INIT: DX feature level B000
[20:52:44] INIT: DX init succeeded
[20:53:31] CORE: Requesting thread creation
[20:53:31] CORE: Creating threads
[20:53:31] CORE: Started control thread, id 33 active 1
[20:53:31] CORE: Started thread 'NativeTrainer.asi' (0x00007FFD10D8A3C0), id 34 active 1
[20:53:31] CORE: Started thread 'RealVR.asi' (0x00007FFD0242A580), id 35 active 1
[20:53:31] CORE: Launching main() for 'NativeTrainer.asi' (0x00007FFD10D8A3C0), id 34
[20:53:31] CORE: Launching main() for 'RealVR.asi' (0x00007FFD0242A580), id 35
```

Both of you guys are completely missing the "CORE" section that actually launches the scripts :-(

**Comment by emozio666** [comment]
> Something is going wrong with your Script Hook V, and I have no idea what it could be. It shouldn't depend on the mod, because the problem happens before my `RealVR.asi` file is even loaded. See how the log looks for me:
> 
> ```
> // GTA V SCRIPT HOOK (build Jul 23 2021, v1.0.2372.0)
> //     (C) Alexander Blade 2015-2021
> [20:52:31] INIT: Started
> [20:52:31] INIT: Success, game version is VER_1_0_2372_0_STEAM
> [20:52:31] INIT: Registering script 'NativeTrainer.asi' (0x00007FFD10D8A3C0)
> [20:52:31] INIT: Registering script 'RealVR.asi' (0x00007FFD0242A580)
> [20:52:39] INIT: Pool 1 extended
> [20:52:39] INIT: Pool 2 extended
> [20:52:39] INIT: Pool 3 extended
> [20:52:39] INIT: Pool 4 extended
> [20:52:44] INIT: GtaThread collection size 189
> [20:52:44] INIT: wnd proc 0x00000000FFFF06CF
> [20:52:44] INIT: IDXGISwapChain 0x00007FFCD8C8D990 (0x00007FFCD8A75920)
> [20:52:44] INIT: IDXGISwapChain set
> [20:52:44] INIT: DX init started
> [20:52:44] INIT: DX feature level B000
> [20:52:44] INIT: DX init succeeded
> [20:53:31] CORE: Requesting thread creation
> [20:53:31] CORE: Creating threads
> [20:53:31] CORE: Started control thread, id 33 active 1
> [20:53:31] CORE: Started thread 'NativeTrainer.asi' (0x00007FFD10D8A3C0), id 34 active 1
> [20:53:31] CORE: Started thread 'RealVR.asi' (0x00007FFD0242A580), id 35 active 1
> [20:53:31] CORE: Launching main() for 'NativeTrainer.asi' (0x00007FFD10D8A3C0), id 34
> [20:53:31] CORE: Launching main() for 'RealVR.asi' (0x00007FFD0242A580), id 35
> ```
> 
> Both of you guys are completely missing the "CORE" section that actually launches the scripts :-(

do we need a complete reinstallation?

**Comment by LukeRoss00** [comment]
I'm not sure. Do you have the game set to auto-load Story Mode? Or do you select it manually each time?

**Comment by emozio666** [comment]
> I'm not sure. Do you have the game set to auto-load Story Mode? Or do you select it manually each time?

Manually

**Comment by LukeRoss00** [comment]
Also, could you try temporarily deleting RealVR.asi? What normally happens if you do that is, you only get the HUD in the headset, without the game world. But it would be interesting to see whether that allows Script Hook V to proceed to the CORE section

**Comment by LukeRoss00** [comment]
> Manually

That might be the difference, let me try it (I always have it on auto)

**Comment by LukeRoss00** [comment]
Nope, it's not that. It still works correctly for me even if I set the Landing Page to On, which at the next run makes the game wait until I load Story Mode manually (which succeeds anyway)

**Comment by emozio666** [comment]
> Also, could you try temporarily deleting RealVR.asi? What normally happens if you do that is, you only get the HUD in the headset, without the game world. But it would be interesting to see whether that allows Script Hook V to proceed to the CORE section

I will try it but unfortunately on the weekend. Thank you for your work and concern.

**Comment by rikiwelche** [comment]
Hello Luc,
I plan to test an installation of your mode on a clean version of the game, I made an original copy of the game in case, I would see if the problem comes from scripthook or installed modes (vr+ nve) because I had installed vehicle packs that could be the cause of the problem 
I’ll tell you how that works.

**Comment by leviofan91** [comment]
I have the same issue on social club clean verison(

**Comment by LukeRoss00** [comment]
@leviofan91 Are you also missing the CORE section in the Script Hook V log?

**Comment by rikiwelche** [comment]
leviofan91 have you natural vision installed?

**Comment by emozio666** [comment]
> Also, could you try temporarily deleting RealVR.asi? What normally happens if you do that is, you only get the HUD in the headset, without the game world. But it would be interesting to see whether that allows Script Hook V to proceed to the CORE section

Report, the same crash without RealVR.asi.
[ScriptHookV.log](https://github.com/LukeRoss00/gta5-real-mod/files/6911698/ScriptHookV.log)

**Comment by LukeRoss00** [comment]
Still missing the CORE section (Script Hook V is not doing its job properly). Perhaps you should verify the game files, something might have gone wrong during the update

**Comment by emozio666** [comment]
> Still missing the CORE section (Script Hook V is not doing its job properly). Perhaps you should verify the game files, something might have gone wrong during the update

This is what I did first, Steam wrote all the files is ok.

**Comment by LukeRoss00** [comment]
Then I'm afraid I'm out of ideas :-( I've never seen this problem before. I believe you might need to reinstall from scratch (make a full backup beforehand if you have enough disc space, so you'll be able to understand what files were changed after reinstalling)

**Comment by leviofan91** [comment]
Game still crashes without RealVR( I don't use natural vision or any other mods.

[10:02:01] INIT: Started
[10:02:01] INIT: Success, game version is VER_1_0_2372_0_NOSTEAM
[10:02:01] INIT: Registering script 'NativeTrainer.asi' (0x00007FFC4EBAA3C0)
[10:02:01] INIT: Registering script 'snow.asi' (0x00007FFC4F6C1C00)
[10:02:01] INIT: Registering script 'timecyclemods.asi' (0x00007FFC48A83D30)
[10:02:01] INIT: Registering script 'RealVR.asi' (0x00007FFC373DA350)
[10:02:09] INIT: Pool 1 extended
[10:02:09] INIT: Pool 2 extended
[10:02:09] INIT: Pool 3 extended
[10:02:09] INIT: Pool 4 extended
[10:02:15] INIT: GtaThread collection size 189
[10:02:15] INIT: wnd proc 0x00000000FFFF0351
[10:02:15] INIT: IDXGISwapChain 0x00007FFC2637D990 (0x00007FFC26165920)
[10:02:15] INIT: IDXGISwapChain set
[10:02:15] INIT: DX init started
[10:02:15] INIT: DX feature level B000
[10:02:15] INIT: DX init succeeded

**Comment by LukeRoss00** [comment]
> without RealVR( I don't use natural vision or any other mods

I'm not sure what you mean. The log you pasted shows that you have NativeTrainer.asi, snow.asi, timecyclemods.asi, RealVR.asi installed

**Comment by leviofan91** [comment]
This is the only log I have since I started. It doesn't change if I delete realvr.  When I start the game I see menu in my HMD, but if I press "story" game crashes after loading screen just when must load(

**Comment by leviofan91** [comment]
after reinstalling game and mod it still doesnt work and now it tells thaqt wrong script hoook version is used(

**Comment by emozio666** [comment]
> after reinstalling game and mod it still doesnt work and now it tells thaqt wrong script hoook version is used(

make sure you download the latest script hook version from July 23rd.

**Comment by zlxxlmbie** [comment]
I just had this issue this morning, after you uninstall the game go into your gtaV folder and clear out EVERYTHING. Then do a fresh reinstall. Uninstalling the game will not remove any game mod folders. Ran just fine after reinstalling the game to the empty folder and redoing the install bat for the mod.

**Comment by leviofan91** [comment]
After I completely remove the game folder and redo all the proccess again it finally works! Thanks a lot guys!!

**Comment by LukeRoss00** [comment]
@emozio666 and @rikiwelche, did removing the game folder completely and reinstalling solve the problem for you too? Should I close the issue?

**Comment by emozio666** [comment]
> @emozio666 and @rikiwelche, did removing the game folder completely and reinstalling solve the problem for you too? Should I close the issue?

I have not tried it yet, but if it helped others, then it will help me too.

**Comment by rikiwelche** [comment]
hello i have not tried too, i will tell you if it works soone as i have time to try

**Comment by munkyonline** [comment]
I had exactly the same problem, tried game repair, upgrading drivers etc but didnt help. I tried deleting the mods folder, and then the game launches but with no mods. I then searched online and found: https://forums.gta5-mods.com/topic/20612/how-to-keep-work-mods-in-update-rpf-after-update/8 which says about replacing the update.rpf file in /mods/update with the one in the /update folder, and either reinstalling your mods, or carefully copying all the files back over.

I've now reinstalled the REAL mod and its loading fine. No need to download/reinstall the game again.

I've not reinstalled Make Visuals Great Again yet. Will report back if installing that again causes any problems :)
